spring-integration-core/src/main/java/org/springframework/integration/config/CorrelationStrategyFactoryBean.java

```diff
@@ -39,8 +39,10 @@
  */
 public class CorrelationStrategyFactoryBean implements FactoryBean<CorrelationStrategy>, InitializingBean {
 
+	@SuppressWarnings("NullAway.Init")
 	private Object target;
 
+	@SuppressWarnings("NullAway.Init")
 	private String methodName;
```

Comment:
This can be null according to the class logic.
While for the `target` we need to ensure that it is not-null.